values.md

```diff
@@ -32,7 +32,7 @@ While we believe that we are building something exceptional, that knowledge does
 
 ### We optimize for the long-term.
 
-While we pursue the mission with intense focus, we recognize that creating the change in the world that we wish to see will not happen overnight. We think, plan, and act for the long-term.
+We recognize that creating the change in the world that we wish to see will not happen overnight. Earning the right to pursue this work over decades requires both long-term vision and near-term execution, both user love and commercial success.
```

Comment:
There is a similar thread [over here](https://github.com/dbt-labs/corp/pull/69#discussion_r1012927548), but I wonder if mentioning "user love" and "commercial success" here is sort of doing the work of putting the values in conversation with each other, that should best be left as the exercise of living out our values. 

e.g., "Users are our best advocates" and "Profits fund our mission"  (and all our other values) - they inform how we execute on our long-term orientation.

That said - I am not totally against that last "user love and commercial success" phrase being in here. They are definitely Good Things™️ and help color in this value. But they are only 2  of many things that will be required to earn the right to pursue this work over decades.